src/v2/Apps/Artist/Routes/Works/index.tsx

```diff
@@ -23,10 +24,11 @@ export const WorksRoute: React.FC<WorksRouteProps> = props => {
   return (
     <>
       <Box>
-        <ArtistCollectionsRail
-          artistID={artist.internalID}
-          includeTopSpacer={false}
```

Comment:
We no longer need this boolean bc the collections rail is no longer the first component on the page. Also this logic caused a bug on artist pages that didn't have collection rails.

<img width="1188" alt="Screen Shot 2020-06-01 at 6 25 03 PM" src="https://user-images.githubusercontent.com/10385964/83460801-45d48900-a435-11ea-9bc2-b1976bdc2352.png">